packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboard.ts

```diff
@@ -230,7 +120,32 @@ export const useGridKeyboard = (
         logger.debug('Multiple Select key pressed');
         onMultipleKeyChange(true);
       }
+    },
+    [logger, onMultipleKeyChange],
+  );
 
+  const onKeyUpHandler = React.useCallback(
+    (event: KeyboardEvent) => {
+      if (isMultipleKey(event.key)) {
+        logger.debug('Multiple Select key released');
+        onMultipleKeyChange(false);
+      }
+    },
+    [logger, onMultipleKeyChange],
+  );
+
+  const onFocusOutHandler = React.useCallback(
```

Comment:
Same as the other one

```suggestion
  const handleFocusOut = React.useCallback(
```